recipes/argon2/all/test_package/argon2-test.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
```

Comment:
Ideally, this test_package would be a C source to check whether argon2 can really be used by a C-only project and didn't require the c++ STL.

Reply:
Ok

Reply:
If possible, please use a common name like `test_package.c`

Reply:
Ok, Its done